metadata/contributors.json

```diff
@@ -1369,7 +1369,8 @@
       "name" : "Byungseon(Sun) Shin",
       "nicks" : [
          "xingri"
-      ]
+      ],
+      "status" : "committer"
```

Comment:
```suggestion
      ],
      "github" : "xingri",
      "status" : "committer"
```

Reply:
@aprotyas  Thanks you for the feedback. Now I fully understood.